### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by Naveen-Sooryaa** [anirudhmalik/xhunter#77] [Q&A]
Give us new update. Inn this version I can't use the whatsapp payload. It's showing gb whatsapp update. Please release new update of this. Love this work.

**Comment by hunte1r** [comment]
I think the programmer left us

**Reply by Annxhilator** [reply]
What do you mean by that?

**Comment by anirudhmalik** [comment]
Closing in favor of v2. See [v2.0-demo](https://github.com/anirudhmalik/xhunter/releases/tag/v2.0-demo) and the repo README. New bugs: open a fresh issue with repro steps. Thanks.
